### PR TITLE
Removing actions from GCP offers

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

File: gcp/shared/group.py [hunk 1]
```diff
@@ -20,21 +20,21 @@ def GenerateConfig(context):
     groupWaiter = GenerateGroupWaiterConfig(context, runtimeconfigName, instanceGroupManagerName, instanceGroupTargetSize)
     groupWaiterName = groupWaiter['name']
 
-    externalIpReadAction = GenerateExternalIpReadActionConfig(context, runtimeconfigName, externalIpCreateActionName, groupWaiterName)
-    externalIpReadActionName = externalIpReadAction['name']
+    # externalIpReadAction = GenerateExternalIpReadActionConfig(context, runtimeconfigName, externalIpCreateActionName, groupWaiterName)
```
Comment (patrick-stephens) [inline gcp/shared/group.py:23]: Why not just delete?

File: gcp/shared/group.py [hunk 2]
```diff
@@ -20,21 +20,21 @@ def GenerateConfig(context):
...
+    # externalIpReadActionName = externalIpReadAction['name']
 
     config={}
     config['resources'] = [
         externalIpCreateAction,
         instanceTemplate,
         instanceGroupManager,
         groupWaiter,
-        externalIpReadAction
+        # externalIpReadAction
```
Comment (patrick-stephens) [inline gcp/shared/group.py:32]: Delete?

File: gcp/shared/group.py [hunk 3]
```diff
@@ -60,25 +60,25 @@ def GenerateExternalIpCreateActionConfig(context, runtimeconfigName):
...
-        'properties': {
-            'name': 'projects/%s/configs/%s/variables/%s' % (project, runtimeconfigName, externalIpVariablePath),
-            'newerThan': '$(ref.%s.updateTime)' % externalIpCreateActionName
-        },
-        'metadata': {
-            'dependsOn': [groupWaiterName]
-        }
-    }
-    return action
+# def GenerateExternalIpReadActionConfig(context, runtimeconfigName, externalIpCreateActionName, groupWaiterName):
```
Comment (patrick-stephens) [inline gcp/shared/group.py:63]: Delete?

File: gcp/shared/group.py [hunk 4]
```diff
@@ -231,6 +231,13 @@ def GenerateStartupScript(context, runtimeconfigName):
...
 NODE_COUNT={node_count}
 
+# Before we install.  we need to specify the cluster host dns to the deployment
+gcp_hostname=$(curl -H "Metadata-Flavor: Google" -s http://metadata/computeMetadata/v1/instance/hostname)
+if [[ "$CLUSTER_HOST" == "$gcp_hostname" ]]; then
+    external_ip=$(curl -H "Metadata-Flavor: Google" -s http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)
+    gcloud beta runtime-config configs variables set external-ip/clusters/{cluster}/groups/server  "$external_ip" --config-name={config}
+fi
+
 if [[ ! -e "couchbase_installer.sh" ]]; then
```
Comment (patrick-stephens) [inline gcp/shared/group.py:241]: Do we want to update this anyway?

**Review by patrick-stephens — APPROVED** [review]
Looks fine but maybe delete the commented out bits?

File: gcp/shared/group.py (hunk 1 above)
Comment (malscent) [reply gcp/shared/group.py:23]: Wanted to show the changes more explicitly.  I've removed the code in the next I am going to be making after this one.

File: gcp/shared/group.py (hunk 2 above)
Comment (malscent) [reply gcp/shared/group.py:32]: Same as above,  next PR will remove the code

File: gcp/shared/group.py (hunk 4 above)
Comment (malscent) [reply gcp/shared/group.py:241]: This is just updating the "output" of the deployment.  So we only want to set it "one time" and from the rally node.

**Pull request merged**